style.css

```diff
@@ -466,3 +486,132 @@ textarea {
   flex-direction: column;
   align-items: center;
 }
+
+/* desktop devices */
+@media only screen and (min-width: 992px) {
+  .status-bar {
+    display: none;
+  }
+
+  .logo {
+    font-size: 18px;
+    line-height: 20px;
+  }
+
+  .navlink {
+    display: flex;
+    gap: 32px;
+  }
+
+  .primary-menu {
+    color: #344563;
+    font-family: "Poppins", Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .background h1 {
+    color: #091e42;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-size: 48px;
+    font-weight: bold;
+    line-height: 60px;
+    display: flex;
+    align-items: center;
+    letter-spacing: 0.37px;
+  }
+
+  .intro {
+    color: #344563;
+    font-family: "Poppins", sans-serif;
+    font-weight: normal;
+    font-size: 20px;
+    display: flex;
+    line-height: 28px;
+  }
+
+  .card-body {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .card-img {
+    width: 544px;
+    height: 448px;
+  }
+
+  .card {
+    display: grid;
+    justify-content: center;
+    grid-template-columns: auto auto;
+    padding: 24px;
+  }
+
+  .invert {
+    order: -1;
+  }
+
+  .about {
+    display: grid;
+    justify-content: center;
+    grid-template-columns: auto auto;
+    padding: 10%;
+  }
+
+  .about-desk {
+    float: left;
+    width: 100%;
+    margin-right: 20px;
+  }
+
+  .about-desk h1 {
+    font-family: "Poppins", Arial, Helvetica, sans-serif;
+    line-height: 52px;
+    font-style: normal;
+    font-weight: bold;
+    color: #091e42;
+  }
+
+  .desk-about-section {
+    display: flex;
+    width: 100%;
+  }
+
+  .java {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+  }
+
+  .language {
+    width: 100%;
+  }
+
+  .language ul {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .contact {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    padding: 10% 25%;
+  }
+
+  footer {
+    padding: 79px 74px 48px 74px;
+  }
+
+  form div {
+    display: flex;
+    justify-content: center;
+  }
+}
```

Comment:
- [ ] The mobile screen is distorted, I think you should fix it before you submit for another review

Reply:
@descholar-ceo Thanks a lot for the review. I'd appreciate it if you would be kind enough to show a screenshot of the distorted mobile screen. it would help a lot